src/ruby/ext/grpc/ext-export-truffleruby.clang

```diff
@@ -1,2 +1 @@
 _Init_grpc_c
-_rb_tr_abi_version
```

Comment:
This doesn't seem correct as it will break the gem on TruffleRuby if I read the PR correctly. TruffleRuby has rb_tr_abi_version before RUBY_VERSION 3.2, specifically it's 3.0 in the 22.3 release currently for TruffleRuby and it already had it before that. We can consider TruffleRuby always had this symbol, versions before are not supported anymore.